build-system/pr-check/build-targets.js

```diff
@@ -170,6 +170,7 @@ const targetMatchers = {
     }
     return (
       file == 'build-system/tasks/check-links.js' ||
+      file.startsWith('build-system/tasks/markdown-toc/') ||
```

Comment:
One more change in this file: In addition to running `gulp markdown-toc` when `tasks/markdown-toc/` is changed, you also want to run the `ava` tests for this directory. 

To do so, add this new condition...

```js
file.startsWith('build-system/tasks/markdown-toc/') ||
```

...to this target matcher...

https://github.com/ampproject/amphtml/blob/5799d67b24c84d78f6a320528ea85583f5724402/build-system/pr-check/build-targets.js#L123-L132